integrations/atlassian/jira/webhooks.go

```diff
@@ -84,39 +82,17 @@ type webhookListResponse struct {
 // https://developer.atlassian.com/server/jira/platform/webhooks/
 func getWebhook(ctx context.Context, l *zap.Logger, baseURL, token string) (int, bool) {
 	// TODO(ENG-965): Support pagination.
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, baseURL+"/rest/api/3/webhook", nil)
+	u := baseURL + "/rest/api/3/webhook"
+	resp, err := common.HTTPGet(ctx, u, "Bearer "+token)
```

Comment:
Maybe have an `HTTPClient` in `common` with a `Get` method (also `Post` ...) and `Do`? (like the `http.DeaultClient`)